engines/python/setup/djl_python/transformer-neuronx.py

```diff
@@ -47,7 +46,7 @@ def convert_opt(self, amp):
         self.model = AutoModelForCausalLM.from_pretrained(
             self.model_id_or_path, low_cpu_mem_usage=True)
         logging.info(f"Start model conversion to INF2 format...")
-        load_path = os.path.join(tempfile.gettempdir(), self.model_id_or_path)
+        load_path = os.path.join("/tmp/tn_model", self.model_id_or_path)
```

Comment:
Why hard code /tmp?

Reply:
For saving to a static place in temp directory.

Reply:
I think tempfile.gettempdir() is better than hard code /tmp. if you want use a subfolder, you can use:

```
load_path = os.path.join(tempfile.gettempdir(), "tn_model",  self.model_id_or_path)
```

Reply:
In S3 downloading, the model id or path is already like /tmp/download4577754, this concat failed

Reply:
Actually, we should respect do the same as how we download the model:
1. try env var: SERVING_DOWNLOAD_DIR first
2. fallback to tempfile.gettempdir()

Reply:
> In S3 downloading, the model id or path is already like /tmp/download4577754, this concat failed

then we should get the file name from model_id_or_path instead of full path

Reply:
I found a better way